Sourcery/Sourcery.swift

```diff
@@ -318,6 +320,18 @@ extension Sourcery {
 
         track("Generating code...", terminator: "")
         status = ""
+        
+        if autoLock && outputPath.exists {
+            do { try output.setImmutable(false) } catch { track("\(error)") }
```

Comment:
I've tested and it works, the idea is: we unlock the file, write to it and than lock it again